fedimint-core/src/lib.rs

```diff
@@ -22,6 +22,7 @@ pub use crate::core::server;
 use crate::encoding::{Decodable, DecodeError, Encodable};
 use crate::module::registry::ModuleDecoderRegistry;
 
+#[cfg(not(target_family = "wasm"))]
```

Comment:
Eventually I assume we'll also want the admin client to be WASM compatible? But I'm ok with getting it ready in a separate PR if it's too big.